examples/copr/setup.cfg

```diff
@@ -1,3 +1,3 @@
 [build_manpages]
 manpages =
-    copr-cli.1:module=copr_cli.main:function=setup_parser
+    copr-cli.1:module=copr_cli.main:function=setup_parser:prog=copr
```

Comment:
Can you please drop this `prog=copr` so we test the implicit behavior (prog is correctly detected from the file name)?  It should still work.